newrelic/structures_newrelic_one_dashboard.go

```diff
@@ -253,11 +253,11 @@ func expandDashboardPageInput(d *schema.ResourceData, pages []interface{}, meta
 				if err != nil {
 					return nil, err
 				}
-
-				widget.Configuration.LogTable, err = expandDashboardLogTableWidgetConfigurationInput(v.(map[string]interface{}), meta)
+				widget.RawConfiguration, err = expandDashboardWidgetRawConfigurationInput(v.(map[string]interface{}), meta)
 				if err != nil {
 					return nil, err
 				}
+				widget.Visualization.ID = "logger.log-table-widget"
```

Comment:
Aha! I was wondering where this special case would finally appear, I knew it had to be somewhere.